profiles/local.yaml

```diff
@@ -5,27 +5,27 @@ experiment:
 
 # Set the registry server URI. This property is especially useful if you have a registry
 # server that’s different from the tracking server.
-# model_registry:
-#   uri: sqlite:////tmp/registry.db
+model_registry:
+  # registry_uri: "sqlite:////tmp/registry.db"
+  # FIXME::REQUIRED: Specifies the name of the Registered Model to use when registering a trained
+  #                 model to the MLflow Model Registry.
+  model_name: "taxi_fare_regressor"
+  # model_uri: "models/model.pkl"
 
-# Use a small sample from the TLC Trip Record Dataset for exploring the MLflow Regression Pipeline
-# (https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page)
-INGEST_DATA_LOCATION: "./data/sample.parquet"
-# Use a larger section of the TLC Trip Record Dataset for training higher-fidelity models
-# INGEST_DATA_LOCATION: "https://nyc-tlc.s3.amazonaws.com/trip+data/yellow_tripdata_2022-01.parquet"
-# Specify the format of the dataset
-INGEST_DATA_FORMAT: parquet
-# Use a small sample from the TLC Trip Record Dataset for exploring the batch scoring feature of the
-# MLflow Regression Pipeline
-# (https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page)
-INGEST_SCORING_DATA_LOCATION: "./data/sample.parquet"
-# Use a larger section of the TLC Trip Record Dataset for the batch scoring feature
-# INGEST_SCORING_DATA_LOCATION: "https://nyc-tlc.s3.amazonaws.com/trip+data/yellow_tripdata_2022-01.parquet"
-# Specify the format of the dataset
-INGEST_SCORING_DATA_FORMAT: parquet
 # Override the default train / validation / test dataset split ratios
 SPLIT_RATIOS: [0.80, 0.10, 0.10]
-# Specify the output location of the batch scoring predict step
-SCORED_OUTPUT_DATA_LOCATION: "./data/sample_output.parquet"
-# Specify the output format of the batch scoring predict step
-SCORED_OUTPUT_DATA_FORMAT: parquet
\ No newline at end of file
+
+INGEST_CONFIG:
+    # For different options please read: https://github.com/mlflow/mlp-regression-template#ingest-step
+    using: parquet
```

Comment:
So here, I'd also recommend to add a layer `input`. First of all, it's more consistent with the rest of steps. Secondly, in the future if we chose to support multiple source types (which I'm fairly confident the day will come), we have the option to make the `input` a list, like below:
```
INGEST_CONFIG:
  input:
    using: parquet
    location: "./data/sample.parquet"
  input:
    using: spark_sql
    sql: "..."
```
@mengxr

Reply:
To further illustrate this future compatibility, see the 2 example below, one with a single source type, one with future extension to support multiple source types. Both should work in the imaginary future version. The first one works right now.

<img width="1006" alt="Screen Shot 2022-10-26 at 3 01 21 PM" src="https://user-images.githubusercontent.com/78067366/198147032-7838768f-9182-4ed6-83e7-3551fa9e06a8.png">

<img width="1006" alt="Screen Shot 2022-10-26 at 3 01 05 PM" src="https://user-images.githubusercontent.com/78067366/198147042-2b2a5f5b-d0a6-4ecf-ba9d-ca99fb27ee1f.png">

Reply:
We don't need `input` because `ingest` already means that, which is different from `batch_score`. If we need to support multiple sources, we can do

~~~
ingest:
  - name: training
     using: parquet
     location: ...
  - name: text_embedding
    using: spark_sql
    sql: ...
~~~